src/js/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * @name Breadcrumbs
+ *
+ * @description
+ *  A breadcrumbs component that will render the appropriate styles for breadcrumbs.
+ *
+ *
+ * @example
+ *  <Breadcrumbs breadcrumbs={breadcrumbs} home={home} />
+ *
+ *
+ */
+
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import Icon from "../Icon/Icon";
+
+const propTypes = {
+  className: PropTypes.string,
+  breadcrumbs: PropTypes.array.isRequired,
+  home: PropTypes.any,
```

Comment:
let's change this to capture the shape of the home object.

Something like:

```
PropTypes.shape({
  url: PropTypes.string
})
```

Reply:
Done!